src/presentational/entry/Entry.jsx

```diff
@@ -0,0 +1,36 @@
+import React, {Component} from 'react' 
+import './Entry.css';
+import Util from '../../Util';
+export default class Entry extends Component {
```

Comment:
the entry represents a food a user had in a specific amount, we display the entries the user added for him to track what they had.